advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>Same in these script-execution cells for building and pushing Docker images. Since you set Python variables like <code>region</code>, etc. below, you could consider running those cells first, then inject the <code>{region}</code> argument into this bash command.</p><p> </p><p><code>! ./container-optimized/build_tools/build_and_push.sh {region}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes this makes a lot of sense. I considered it, but interspersing SageMaker Python code with setup shell scripts appeared less clean. This is because the setup code needs to be done only once and may be skipped at all subsequent walk throughs of the notebook.

Reply:
`region` and `s3_bucket` variables were added in recent revision. Thanks!